test/e2e_node/g8r.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make bucket and a folder for e2e-node test logs.
+# Populate the folder from the logs stored in /tmp/_artifacts/ in the same way as a
+# jenkins build would, and then print the URL to view the test results on Gubernator
+
+bucket_name="${USER}-g8r-logs"
+echo ""
+echo "Using bucket ${bucket_name}"
+gsutil mb gs://${bucket_name}/
+
+GCS_JOBS_PATH="gs://${bucket_name}/logs/e2e-node"
+
+TMP_LOG_PATH="/tmp/_artifacts/"
+BUILD_LOG_PATH="/tmp/build-log.txt"
+readonly gcs_acl="public-read"
+
+if [[ -e "build-log.txt" ]]; then
+	echo "Moving build-log.txt"
+	mv build-log.txt /tmp/
+fi
+
+# Keep build number updated
+if ! gsutil -q stat "${GCS_JOBS_PATH}/latest-build.txt"; then
+	BUILD_NUMBER=1
+else
+	BUILD_NUMBER=$(gsutil cat "${GCS_JOBS_PATH}/latest-build.txt")
+	let "BUILD_NUMBER += 1"
```

Comment:
I'm trying to think of a more robust way of doing this. I think the ideal would be to use a date/time stamp. I think you could use the mtime of the build-log to get the time of the run. This would have two advantages: 1) easier to identify the run in the GCS bucket, and 2) If I want to review the logs again, I can rerun the script and it won't re-upload everything.